apps/emqx_prometheus/rebar.config

```diff
@@ -3,7 +3,7 @@
 {deps, [
     {emqx, {path, "../emqx"}},
     {emqx_utils, {path, "../emqx_utils"}},
-    {prometheus, {git, "https://github.com/deadtrickster/prometheus.erl", {tag, "v4.8.1"}}}
+    {prometheus, {git, "https://github.com/emqx/prometheus.erl", {tag, "v4.10.0-emqx-1"}}}
```

Comment:
such version numbers will be tricky in the future to support hot-upgrade.
maybe use a fourth digit instead